site/src/pages/HealthPage/DatabasePage.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { StoryObj, Meta } from "@storybook/react";
+import { DatabasePage } from "./DatabasePage";
+import { HealthLayout } from "./HealthLayout";
+import {
+  reactRouterOutlet,
+  reactRouterParameters,
+} from "storybook-addon-react-router-v6";
+import { useQueryClient } from "react-query";
+import { MockHealth } from "testHelpers/entities";
+
+const meta: Meta = {
+  title: "pages/Health/Database",
+  render: HealthLayout,
+  parameters: {
+    layout: "fullscreen",
+    reactRouter: reactRouterParameters({
+      routing: reactRouterOutlet(
```

Comment:
Just realized something: where is the context value from `useOutletContext` coming from for these nested page stories? Is it defined here?

Reply:
It is defined in the HealthPageLayout component